### DISCUSSION TITLE
Welcome to capacitor-googlemaps-native Discussions!

### DISCUSSION THREAD

**Discussion by hemangsk** [capacitor-community/google-maps#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hiro5id** [comment]
What does WIP mean in the table of the reame?

**Reply by hemangsk** [reply]
It means _work in progress_

**Reply by hemangsk** [reply]
to be clear, Initially I added WIP in all those fields where I thought me or a contributor would get it done at some point in time.

**Comment by selected-pixel-jameson** [comment]
@hemangsk I have two questions.

- Are you accepting pull-requests? 
- Is there a reason that this plugin isn't under the capacitor community plug-ins?

**Reply by hemangsk** [reply]
> @hemangsk I have two questions.
> 
> * Are you accepting pull-requests?

Yes, definitely :)

> * Is there a reason that this plugin isn't under the capacitor community plug-ins?

I've posted on capacitor slack channel last week regarding this and I'm hoping that the porting happens soon.

**Reply by selected-pixel-jameson** [reply]
Awesome!

**Comment by S0NUKUMAR** [comment]
Hello team , 

can you one suggest what i am doing wrong , i have seen my code couple of times , no error found , but not able to view the Map in the system , getting below error .

Error: Method not implemented.
    at CapacitorGoogleMapsWeb.create (web.js:10:1)
    at index.js:196:35
    at _ZoneDelegate.push.3279._ZoneDelegate.invoke (zone.js:409:1)
    at Object.onInvoke (core.mjs:25476:33)
    at _ZoneDelegate.push.3279._ZoneDelegate.invoke (zone.js:408:1)
    at Zone.push.3279.Zone.run (zone.js:169:1)
    at zone.js:1326:1
    at _ZoneDelegate.push.3279._ZoneDelegate.invokeTask (zone.js:443:1)
    at Object.onInvokeTask (core.mjs:25463:33)
    at _ZoneDelegate.push.3279._ZoneDelegate.invokeTask (zone.js:442:1)

**Reply by shahilnissam** [reply]
I am facing the same issue, Can anyone help here? @S0NUKUMAR How you resolved this issue?
